File: pushpop.js
let stack=[]
top=-1
function push(value){
    top=top+1
    stack[top]=value
}
function pop(){
    removed =stack[top]
    top=top-1
    return removed
}
push(4)
push(10)
push(20)
pop(20)
console.log(stack)
console.log(pop())

File: pushpop.java
import java.util.Scanner;
public static void main(String[]args){
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    int[] stack=new int[n];
    int top=-1;
    for(int i=0;i<n;i++){
        int value=sc.nextInt();
        top++;
        stack[top]=value;
    }
    System.out.println(stack[top]);
    stack[top].pop();
    top--;
    System.out.println(stack[top]);
}